code/datums/components/aiming.dm

```diff
@@ -0,0 +1,167 @@
+// Aiming component, ported from NSV
+
+/datum/component/aiming
+	can_transfer = FALSE
+	var/mob/living/user = null
+	var/mob/living/target = null
+	var/aiming_cooldown = FALSE
+	var/cooldown_time = 5 SECONDS // So you can't spam aiming for faster bullets/spamming lines
```

Comment:
Use our cooldown macros instead

Reply:
Changed it to use the macros, let me know if I forgot anything.